docs/user/reference/general/initializing.md

```diff
@@ -56,22 +56,25 @@ May only be called once. Subsequent calls to `initialize` are no-op.
-- `appBuild`: A build identifier e.g. the build identifier generated by a CI system (e.g. "1234/A"). If not present, `app_build` will be reported as "Unknown" on all pings [`client_info`](../../user/pings/index.html#the-client_info-section) section.
-- `appDisplayVersion`: The user visible version string for the application running Glean. If not present, `app_display_version` will be reported as "Unknown" on all pings [`client_info`](../../user/pings/index.html#the-client_info-section) section.
-- `serverEndpoint`: The server pings are sent to. Defaults to `https://incoming.telemetry.mozilla.org`.
-- `maxEvents`: The maximum number of events the Glean storage will hold on to before submitting the 'events' ping. Defaults to 1 for Glean.js, 500 for all other SDKs. Refer to the [`events` ping documentation](../../user/pings/events.md) for more information on its scheduling.
-- `httpUploader`: A custom HTTP uploader instance, that will overwrite Glean's provided uploader. Useful
-for users that wish to use specific uploader implementations. See [Custom Uploaders](#custom-uploaders)
-for more information on how and when the use this feature.
-- `logLevel`: The level for how verbose the internal logging is. The level filter options in order from least to most verbose are: `Off`, `Error`, `Warn`, `Info`, `Debug`, `Trace`. See the [`log` crate docs](https://docs.rs/log/latest/log/) for more information.
-- `rateLimit`: Optional.
-  Specifies the maximum number of pings that can be uploaded per interval of a specified number of seconds.
-  Default is to use the SDK default (presently 15 pings per 60s interval).
-- `experimentationId`: Optional. An identifier derived by the application to be sent in all pings for the purpose of experimentation. See the experiments API documentation for more information.
+Note that on some SDKs some of the options are taken as a configuration object.
+Check the respective SDK documentation for details.
+
+| Configuration Option | Default value | Description |
+| -------------------- | ------------- | ----------- |
+| `applicationId` | On Android/iOS: determined automatically. Otherwise *required*. | Application identifier. For Android and iOS applications, this is the id used on the platform's respective app store and is extracted automatically from the application context. |
+| `uploadEnabled` | _Required_ | The user preference on whether or not data upload is enabled. |
+| `appChannel` | - | The application's release channel. When present, the `app_channel` will be reported in all ping's [`client_info`](../../user/pings/index.html#the-client_info-section) section. |
+| `appBuild` | On Android/iOS: determined automatically. Otherwise: - | A build identifier e.g. the build identifier generated by a CI system (e.g. "1234/A"). If not present, `app_build` will be reported as "Unknown" on all pings [`client_info`](../../user/pings/index.html#the-client_info-section) section. |
```

Comment:
```suggestion
| `appBuild` | On Android/iOS: determined automatically. Otherwise: - | A build identifier e.g. the build identifier generated by a CI system (e.g. "1234/A"). If not present, `app_build` will be reported as "Unknown" on all pings' [`client_info`](../../user/pings/index.html#the-client_info-section) sections. |
```

Reply:
(and so forth)